bin/run

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
`/bin/run` and `bin/client_run` files looks 100% equal. Do we need to keep both?

Reply:
We will drop `bin/client_*` and `bin/server_` then single executable will be ready and stable